### PR TITLE
Add cluster details page (1/2)

### PR DESCRIPTION
**Pull request by gkarat**
Fixes https://issues.redhat.com/browse/CCXDEV-4984

Forked from https://github.com/RedHatInsights/ocp-advisor-frontend/pull/6

### CODE REVIEW

File: src/PresentationalComponents/Inventory/InventoryDetails.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import {
+  AppInfo,
+  DetailWrapper,
+  InventoryDetailHead,
+} from '@redhat-cloud-services/frontend-components/Inventory';
```
Comment (AllenBW) [inline src/PresentationalComponents/Inventory/InventoryDetails.js:5]: As this stuff is closely tied with RHEL Inventory/systems, thinkin yer probably not gonna wanta use it for anything (doesn't have clusters 🤔 )

Comment (gkarat) [reply src/PresentationalComponents/Inventory/InventoryDetails.js]: Ah, alright, I thought `inventory` is abstracted from the context of systems/clusters/whatever.

**Comment by gkarat** [comment]
To be continued in https://github.com/RedHatInsights/ocp-advisor-frontend/pull/9

**Comment by gkarat** [comment]
let'ssss merge it? wdyt @AllenBW? 🤩

**Review by AllenBW — APPROVED** [review]
lg2m! 👍

**Pull request merged**